test/e2e_node/memory_manager_test.go

```diff
@@ -781,66 +911,1121 @@ var _ = SIGDescribe("Memory Manager Incompatibility Pod Level Resources", framew
 			if initialConfig.FeatureGates == nil {
 				initialConfig.FeatureGates = make(map[string]bool)
 			}
-			initialConfig.FeatureGates["PodLevelResources"] = true
 		})
 
-		ginkgo.Context("", func() {
-			ginkgo.BeforeEach(func() {
-				// override pod parameters
+		ginkgo.BeforeEach(func() {
+			// override pod parameters
+			ctnParams = []memoryManagerCtnAttributes{
+				{
+					ctnName: "memory-manager-none",
+					cpus:    "100m",
+					memory:  "128Mi",
+				},
+			}
+		})
+
+		ginkgo.JustAfterEach(func() {
+			// reset containers attributes
+			ctnParams = []memoryManagerCtnAttributes{}
+			initCtnParams = []memoryManagerCtnAttributes{}
+		})
```

Comment:
this looks fragile. Why do we need this override dance?
I'd rather inline the relevant code in the tests. Some duplication maybe, but much more isolation and overall the tests are easier to understand and debug. Deduplicating code but having to chase the chain of Before/After to reconstruct the actual state is not much more maintenable.